depthai_nodes/ml/parsers/keypoints.py

```diff
@@ -53,6 +55,7 @@ def __init__(
 
         self.scale_factor = scale_factor
         self.n_keypoints = n_keypoints
+        self._warned = False
```

Comment:
Should we maybe add this as a parameter (but keep the default value to False)? The added value of doing this is that we have to include it in the docstring and explain what it means (the variable name itself isn't really self explanatory).